tests/test_debug.py

```diff
@@ -0,0 +1,108 @@
+import pytest
+
+from substratest import assets
+from substratest.factory import Permissions
+
+
+@pytest.mark.remote_only
+@pytest.mark.slow
+def test_execution_debug(client, debug_client, factory, default_dataset, default_objective):
+
+    spec = factory.create_algo()
+    algo = client.add_algo(spec)
+
+    #  Add the traintuple
+    # create traintuple
+    spec = factory.create_traintuple(
+        algo=algo,
+        dataset=default_dataset,
+        data_samples=default_dataset.train_data_sample_keys[0],
+    )
+    traintuple = debug_client.add_traintuple(spec)
+    assert traintuple.status == assets.Status.done
+    assert traintuple.out_model is not None
+
+    # Add the testtuple
+    spec = factory.create_testtuple(
+        objective=default_objective,
+        traintuple=traintuple,
+        data_samples=default_dataset.test_data_sample_keys[0],
+    )
+    testtuple = debug_client.add_testtuple(spec).future().wait()
+    assert testtuple.status == assets.Status.done
+    assert testtuple.dataset.perf == 3
+
+
+@pytest.mark.remote_only
+@pytest.mark.slow
```

Comment:
The test must be quite fast to run as the execution is local? (same for other tests)

Reply:
As a reference, `test_compute_plan_single_client_success` and `test_compute_plan_update ` take 7.55s to run in "local" mode in the CI. I guess we can expect the tests in test_debug to be of same or longer duration.
I don't have a clear grasp of `slow` in substra tests, what would be the rule of thumb ?

Reply:
Ok, I thought it would be much faster.
I don't think we have a proper rules for tagging a test as slow. The main idea is to ensure the "fast mode" can be used to run the tests for a quick development rules. So tagging a test as slow is a trade-off between use case coverage and performance.

Let's keep the slow tag for these tests then.